css/style-sass-components.css

```diff
@@ -6364,28 +6362,46 @@ Style guide: page-header.5
   .page-header--xlarge-intro, .node-info-homepage.view-mode-homepage_political .page-header {
     height: 320px; }
     .page-header--xlarge-intro .page-header__hero-title, .node-info-homepage.view-mode-homepage_political .page-header .page-header__hero-title {
-      margin: auto;
-      padding-top: 5em;
-      background-color: transparent; }
-    .page-header--xlarge-intro .field__label--above, .node-info-homepage.view-mode-homepage_political .page-header .field__label--above {
-      width: 146px;
-      height: 34px;
-      margin: 0;
-      line-height: 34px;
-      background-color: #003776;
-      text-align: center; }
-    .page-header--xlarge-intro p, .node-info-homepage.view-mode-homepage_political .page-header p {
-      width: 550px;
-      padding: 22px 5px 22px 33px;
-      background-color: #fff;
-      color: #004494; } }
+      margin: 245px 0 0; } }
+
+@media (min-width: 768px) {
+  .page-header--xlarge-intro .page-header__hero-title, .node-info-homepage.view-mode-homepage_political .page-header .page-header__hero-title {
+    margin: auto;
+    padding-top: 5em;
+    background-color: transparent; }
+  .page-header--xlarge-intro .field__label--above, .node-info-homepage.view-mode-homepage_political .page-header .field__label--above {
+    width: 146px;
+    height: 34px;
+    margin: 0;
+    line-height: 34px;
+    background-color: #003776;
+    text-align: center; }
+  .page-header--xlarge-intro p, .node-info-homepage.view-mode-homepage_political .page-header p {
+    width: 550px;
+    padding: 22px 5px 22px 33px;
+    background-color: #fff;
+    color: #004494; } }
+
+.node-info-homepage.view-mode-homepage_political .page-header.page-header--image {
```

Comment:
@pedroraf, do we really want to have something like this in a component?
I wouldn't have too many problems seeing this code in a subtheme, but as a part of the component itself i think it is bad, at the same time i see that we are still showing those "instance specific" demoes in the main styleguide, therefore we need to support them from the theme side, but this should change in my opinion.